biz.aQute.bnd.reporter/src/biz/aQute/bnd/reporter/exporter/ReportExporter.java

```diff
@@ -68,7 +68,7 @@
 		final Map<String, ExportReportInstruction> instructions = getScopedExportInstruction();
 		instructions.entrySet()
 			.forEach(config -> {
-				final File destination = IO.getFile(_processor.getBase(), config.getKey());
+				final File destination = IO.getFile(_processor.getBase(), cleanKey(config.getKey()));
```

Comment:
I think you are looking for `Processor.removeDuplicateMarker(config.getKey())`. Which means you do not need the `cleanKey` method.

Reply:
Thanks!